include/enotify.php

```diff
@@ -106,9 +106,8 @@ function notification($params)
 	}
 
 	if ($params['type'] == NOTIFY_COMMENT) {
-		$p = q("SELECT `ignored` FROM `thread` WHERE `iid` = %d AND `uid` = %d LIMIT 1",
-			intval($parent_id),
-			intval($params['uid'])
+		$p = q("SELECT `ignored` FROM `thread` WHERE `iid` = %d LIMIT 1",
```

Comment:
No intention to convert this to `dba:select` while you're at it?

Reply:
Currently I'm more in the mood to enhance this public item stuff. There is much work to do.

Reply:
Gotcha.